src/components/Content/Breadcrumb.css

```diff
@@ -1,19 +1,21 @@
 @import url('../Helper/Helper.css');
 
 .breadcrumbs {
+  overflow: scroll;
   margin-bottom: 20px;
```

Comment:
You can remove the margin-bottom now, since this has no effect (somehow :D)

Reply:
done